site/pages/[locale]/sign-message.js

```diff
@@ -96,7 +97,11 @@ const SignMessageForm = function () {
         </div>
       )}
 
-      <Button {...signButton}>{t('sign')}</Button>
+      {active ? (
+        <Button {...signButton}>{t('sign')}</Button>
+      ) : (
+        <Wallet cta="true" />
```

Comment:
The logic checking `active`, then showing the "Connect..." or the action could be moved to its own component. Something like:

```js
        <MainCallToAction>
          <Button {...signButton}>{t('sign')}</Button>
        </MainCallToAction>
```

Inside `MainCallToAction`, check `active` and show the children or the wallet connection button. That way, it would be easier to implement this behavior in each app.

Reply:
done